src/components/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+NavBar.propTypes = {
+  onClickSearch: PropTypes.func,
+  onClickHome: PropTypes.func,
+}
+
+export default function NavBar({ onClickSearch, onClickHome }) {
```

Comment:
Yesterday I received feeback for better naming: 
```suggestion
export default function NavBar({ onSearch, onHome }) {
```